### PR TITLE
Fixes #12881 - Add an installer check about the LANG environment vari…

### PR DESCRIPTION
**Pull request by bkearney**
…able

### CODE REVIEW

File: checks/lang.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+INVALID_LANG = %q(The LANG envrionment variable should not be set to C)
+
+def error_exit(message, code)
+  $stderr.puts message
+  exit code
+end
+
+begin
+    error_exit(INVALID_LANG, 1)  if ENV["LANG"] == "C"
```
Comment (ehelms) [inline checks/lang.rb]: 2 space indent

File: checks/lang.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+rescue
+    # Eat the exception and continue
```
Comment (ehelms) [inline checks/lang.rb]: What exception might be thrown here? Looks like the code in the begin statement is a fairly simple check and function call

Comment (bkearney) [inline checks/lang.rb]: Likely none, but this seemed to be the pattern for the other checks. I assumed the logic was these should never fail

**Comment by ehelms** [comment]
I'd say that begin/end is only the format for one check, e.g. https://github.com/Katello/katello-installer/blob/master/checks/hostname.rb#L41-L49

**Comment by bkearney** [comment]
@ehelms removed the extraneous begin/end

**Comment by ehelms** [comment]
ACK

**Pull request merged**